osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -154,6 +154,12 @@ protected override void InitialiseDefaults()
             SetDefault(OsuSetting.IncreaseFirstObjectVisibility, true);
             SetDefault(OsuSetting.GameplayDisableWinKey, true);
 
+            // Replay
+            SetDefault(OsuSetting.ReplayHitMarkersEnabled, false);
+            SetDefault(OsuSetting.ReplayAimMarkersEnabled, false);
+            SetDefault(OsuSetting.ReplayAimLinesEnabled, false);
+            SetDefault(OsuSetting.ReplayCursorHideEnabled, false);
+
```

Comment:
There was a reason I proposed using `[SettingSource]` – the way you've done this, you've polluted the main project with osu!-specific settings.

At very least these should be in the ruleset config manager.

Reply:
oops, wrong config manager 😓

Reply:
@Sheppsu I may look at getting this in the next release, in which case I would take over the final cleanup work to make this happen. Do you have any in-flight changes? I'll probably get to looking at this later today.

Reply:
Oh I missed this message earlier. I've just committed the changes I was working on before I go to sleep.

Reply:
Oh I missed this message earlier. I've just committed the changes I was working on before going to sleep.

Reply:
Okay, can you confirm you're done making changes? If so I'll commit any remaining things I want fixed directly.

Reply:
Yes, go ahead.